lib/circleci/circleci.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// CircleCI workload identity
+//
+// Key values:
+// Issuer of tokens: `https://oidc.circleci.com/org/ORGANIZATION_ID`
+// Audience: `ORGANIZATION_ID`
+//
+// `iat` and `exp` will be included and should be respected.
+//
+// Useful references:
+// - https://circleci.com/docs/openid-connect-tokens/
+
+package circleci
+
+import "fmt"
+
+const IssuerURLTemplate = "https://oidc.circleci.com/org/%s"
+
+func issuerURL(template, organizationID string) string {
```

Comment:
nit: I may have missed another usage, but does `template` need to be a parameter? It seems we only ever use the constant value.

Reply:
Yeah it's used as part of the test to make requests hit the test server instead :)